### PR TITLE
Add capability result viewed

### PR DESCRIPTION
**Pull request by GTcharles**
This PR adds the following Survey events:

- Capability Result Viewed event.

### CODE REVIEW

File: pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module PupilfirstXapi
+  module Objects
+    class CapabilityResultView
+      def call(survey, uri_for)
+        target_uri = uri_for.call(survey)
+
+        Builder.new(
+          id: target_uri,
+          type: 'http://adlnet.gov/expapi/activities/resource',
+          name: "survey #{survey.name}",
```
Comment (mpraglowski) [inline pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb:10]: Why not jus a name?

File: pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module PupilfirstXapi
+  module Objects
+    class CapabilityResultView
```
Comment (mpraglowski) [inline pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb:3]: If you pass survey here and use its names to describe the object then I think this should be just a `Survey` class. This is the object - not the activity.

File: pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      def call(survey, uri_for)
+        target_uri = uri_for.call(survey)
+
+        Builder.new(
+          id: target_uri,
+          type: 'http://adlnet.gov/expapi/activities/resource',
```
Comment (mpraglowski) [inline pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb:9]: I think we could find a better type for this.

Comment (GTcharles) [reply pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb]: That is true. Replaced it with http://id.tincanapi.com/activitytype/survey.
Somehow I couldn't find a fitting one when I was coding this.

File: pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb (hunk 1 above)
Comment (GTcharles) [reply pupilfirst_xapi/lib/pupilfirst_xapi/objects/capability_result_view.rb]: Agreed.

**Pull request merged**